ios.appcache

```diff
@@ -0,0 +1,21 @@
+CACHE MANIFEST
+
+#version 1.0 2-28-2017
+
+CACHE:
+
+# internal HTML documents
+https://caltrainschedule.io/
+
+# style sheets
+https://caltrainschedule.io/stylesheets/default.css
+
+# style sheet images
+"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEgAAABICAQAAAD/5HvMAAAAqklEQVR4Ae3WRwKCQBBE0To5eYYjm22XZjI9hv9rZ3wqqPrKiMhuBggQIECAAAECBAjQ14EA5To+7Bb0eF0uhyrZwFVyqvLnzCdVcq7s5JRKUJmKU6joJvlyys4nKZ44hQdnOKnwPJeqTpIDp5Y9rO4kuXO6SStzGtmbNUpQkHUsJOWkJ0XZgEU51coGrpVDubYPuyU8XpfzfxkQIECAAAECBAgQIEDOEdEFZ84Z0079TZ0AAAAASUVORK5CYII="
```

Comment:
I wonder what this line is about. Is it an error?

Reply:
![image](https://user-images.githubusercontent.com/883126/30388384-6571fc64-9864-11e7-9841-b3ad64d73229.png)

Reply:
Yes I know *what* it is...I want to know why it is in a cache manifest file...it's not a file, it's a string, so what use is it in a cache manifest? Surely if it is in, say, the value of an img tag's src attribute, then it is already there in the code- what's the point of caching it?
My guess is that someone used a script to get the src attribute for img tags, and one of them in hardcoded as a base64 string; but I'm not sure, hence the question.